browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -79,45 +97,48 @@ int OnBeforeURLRequest_StaticRedirectWork(
+          "https://laptop-updates.brave.com/*"),
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://laptop-updates-staging.herokuapp.com/*"),
+          "https://laptop-updates-staging.herokuapp.com/*"),
       // CRX file download
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://brave-core-ext.s3.brave.com/release/*"),
+          "https://brave-core-ext.s3.brave.com/release/*"),
       // Safe Browsing and other files
       URLPattern(URLPattern::SCHEME_HTTPS, "https://static.brave.com/*"),
       // We do allow redirects to the Google update server for extensions we
       // don't support
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://update.googleapis.com/service/update2"),
+          "https://update.googleapis.com/service/update2"),
 
       // Rewards URLs
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://ledger.mercury.basicattentiontoken.org/*"),
+          "https://ledger.mercury.basicattentiontoken.org/*"),
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://balance.mercury.basicattentiontoken.org/*"),
+          "https://balance.mercury.basicattentiontoken.org/*"),
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://publishers.basicattentiontoken.org/*"),
+          "https://publishers.basicattentiontoken.org/*"),
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://publishers-distro.basicattentiontoken.org/*"),
+          "https://publishers-distro.basicattentiontoken.org/*"),
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://ledger-staging.mercury.basicattentiontoken.org/*"),
+          "https://ledger-staging.mercury.basicattentiontoken.org/*"),
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://balance-staging.mercury.basicattentiontoken.org/*"),
+          "https://balance-staging.mercury.basicattentiontoken.org/*"),
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://publishers-staging.basicattentiontoken.org/*"),
+          "https://publishers-staging.basicattentiontoken.org/*"),
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://publishers-staging-distro.basicattentiontoken.org/*"),
+          "https://publishers-staging-distro.basicattentiontoken.org/*"),
 
       // Safe browsing
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://safebrowsing.brave.com/v4/*"),
+          "https://safebrowsing.brave.com/v4/*"),
       URLPattern(URLPattern::SCHEME_HTTPS,
-                 "https://ssl.gstatic.com/safebrowsing/*"),
+          "https://ssl.gstatic.com/safebrowsing/*"),
 
       URLPattern(URLPattern::SCHEME_HTTPS, "https://crlsets.brave.com/*"),
       URLPattern(URLPattern::SCHEME_HTTPS, "https://crxdownload.brave.com/*"),
+
+      // Brave's translation relay server
+      URLPattern(URLPattern::SCHEME_HTTP, kBraveTranslateServerPrefix),
```

Comment:
why SCHEME_HTTP instead of SCHEME_HTTPS?

Reply:
This is for current local server I'm running during development,
this scheme and address will be updated once go-translate is deployed by devops.

Reply:
nvm, i see because it's localhost